compiler/rustc_metadata/src/creader.rs

```diff
@@ -359,7 +359,11 @@ impl<'a> CrateLoader<'a> {
         for (_, other) in self.cstore.iter_crate_data() {
             // Same stable crate id but different SVH
             if other.stable_crate_id() == root.stable_crate_id() && other.hash() != root.hash() {
-                return Err(CrateError::SymbolConflictsOthers(root.name()));
+                span_bug!(
+                    DUMMY_SP, //FIXME:  pass in the correct span here
+                    "Previously returned E0523 here. Root.name() = {}",
```

Comment:
Suggestions for what the span error should say? Is there a standard "we removed this error" message that the compiler emits?

Reply:
Error message can be anything -- this should never be seen by a user, so perhaps make it useful for anyone debugging.